react-hot-button-component/dist/styles.css

```diff
@@ -0,0 +1,50 @@
+
+.button {
+  font-size: 1.25rem;
+  font-weight: 600;
+  font-family: 'Courier New', Courier, monospace;
+  padding: 7px 10px;
+  border: solid black 3px;
+  z-index: 0;
+  transform: scale(1);
+  transition-duration: 800ms;
+}
+
+.buttonShadow {
+  box-shadow: 3px 5px;
```

Comment:
It looks like your box-shadow is still there with other background-colors, it is just harder to see with the darker back grounds. You can give it more of a shadow look by adding a 3rd value for "spread" that will make it look more like a shadow and a transparent color that will also help. So something like:

```CSS
box-shadow: 3px 3px 5px rgba(0, 0, 0, 0.6);
```

You can play around with the values to get the desired effect.